maintenance_equipment_always_fold/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Maintenance Equipment Categories always_fold is indicate it",
+    "summary": "Equipment categories will always be folded if I indicate it "
+    "with an always_fold Boolean; if not, let him behave as before",
+    "author": "Odoo Community Association (OCA), Solvos",
+    "maintainers": ["dalonsod"],
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Equipments",
+    "website": "https://github.com/OCA/maintenance",
+    "depends": ["maintenance"],
+    "data": ["views/maintenance_views.xml", "demo/maintenance_demo.xml"],
```

Comment:
Demo data should go to demo

Reply:
Yes, my fault, thanks for the review

Reply:
Well, it came from history, so my fault as I didn't found it in the first review :wink: